src/inputs/datePickerInput/datePickerInput.jsx

```diff
@@ -107,6 +109,11 @@ const propTypes = {
         PropTypes.shape({ current: PropTypes.any }), // eslint-disable-line react/forbid-prop-types
     ]),
 
+    /**
+     * Hides the calendar picker when introducing the date manually
```

Comment:
maybe "introducing" should be "entering"??

Reply:
jeoffm: Fixed: https://github.com/saddlebackdev/react-cm-ui/pull/458/commits/18bb844efde21bde7a0e9d6516c6013c42418ff8#diff-4d384323ae0810e29196d88f93fb17cd2cda4bdff55a0f241e7558f675d82cc2L113-R114